data/covid19filter-README.txt

```diff
@@ -0,0 +1,45 @@
+This is an export created with Social Feed Manager.
+
+COLLECTION INFORMATION
+Collection name: Coronavirus Hashtags
+
+Collection id: b841df17b3eb49f78d048fe9e53300ac
+Collection set: Coronavirus (collection set id b36187d408d64e14aacdd3b1a9334a34)
+Harvest type: Twitter filter
+
+Harvest options:
+
+Seeds:
+* Track: #Coronavirus, #Coronaoutbreak, #COVID19 - Active
+
+Change log:
```

Comment:
Hi Joe! I'm having trouble understanding what the change log here means. Could you explain what the date/times represent?

Reply:
So this is actually the README from the data itself, so when I downloaded the data this came with it. I didn't mean to keep it here because it's pretty useless. I'm probably just going to delete it.